extensions/wikia/Email/Controller/PasswordResetLinkController.class.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\EmailController;
+use Email\Fatal;
+
+/**
+ * Class PasswordResetLinkController
+ *
+ * @requestParam int targetUserId : The user id to send the password reset link email to
+ * @requestParam string token : The token by which a user will be identified
+ *
+ * @package      Email\Controller
+ */
+class PasswordResetLinkController extends EmailController {
+
+	protected $token;
+	protected $returnUrl;
+	const MAX_LINK_LENGTH = 40;
+
+	// TODO: use real page address, also differentiate between dev and prod envs.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)